### ISSUE TITLE
ArrayIndexOutOfBoundsException on createEntropy method

### ISSUE THREAD

**Issue by dawidkarczewski-dotdata** [thibaultmeyer/cuid-java#8]
**Describe the bug**
After many iterations of creating CUIDv2 by method `CUID.randomCUID2()` an exception is thrown:

```
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index -8 out of bounds for length 10
	at io.github.thibaultmeyer.cuid.CUID$CUIDv2.createEntropy(CUID.java:271)
	at io.github.thibaultmeyer.cuid.CUID.randomCUID2(CUID.java:71)
	at io.github.thibaultmeyer.cuid.CUID.randomCUID2(CUID.java:52)
	at org.example.Main.main(Main.java:8)
```

**To Reproduce**

Just generate CUID in the loop. It takes time (in my case over 1h), for example

```
import io.github.thibaultmeyer.cuid.CUID;

public class Main {
    public static void main(String[] args) {
        while (true) {
            CUID.randomCUID2();
        }
    }
}
```

**Expected behavior**
An exception should never be thrown

**Additional context**
Java 11

**Comment by czarekm** [comment]
Hi @thibaultmeyer . I've created PR to fix it => https://github.com/thibaultmeyer/cuid-java/pull/9

**Comment by thibaultmeyer** [comment]
Resolved by #9 
Version 2.0.3 released
